error-issue/index.ts

```diff
@@ -15,7 +15,44 @@
  */
 
 import express from 'express';
+import statusCodes from 'http-status-codes';
+import {ErrorIssueBot} from './src/bot';
+import {ErrorReport} from './src/types';
 
-module.exports.app = (req: express.Request, res: express.Response) => {
-  res.sendStatus(200);
+const GITHUB_REPO = process.env.GITHUB_REPO || 'rcebulko/amphtml';
```

Comment:
Add TODO to un-default this or change the default to `ampproject/amphtml` 🙃

Reply:
Done.